### PR TITLE
Mark acmeCA-2.0 feature as GA

### LINKED ISSUE #13435: acmeCA-2.0: Enable acmeCA feature for GA
Enable the acmeCA and certificateCreator-2.0 for GA.

For #9017

### PR DESCRIPTION
**Pull request by kristip17**
Fixes #13435 

Enable `acmeCA-2.0` and `certificateCreator-2.0`

Received approval to set edition to `base`

### CODE REVIEW

**Comment by kristip17** [comment]
#build

**Comment by kristip17** [comment]
Running this build, but working with the FAT approval team in parallel.

**Comment by kristip17** [comment]
#libby-bot

**Comment by kristip17** [comment]
This will not be merged until we have FAT approval.

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature:10]: full probably isn't the right edition?

Comment (kristip17) [reply dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature]: It is not! We are reviewing the correct edition.

**Comment by kristip17** [comment]
#run-libby

**Comment by kristip17** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature
Comment (kristip17) [reply dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.certificateCreator-2.0.feature]: Changed to base.

**Comment by kristip17** [comment]
@tevans78  -- ready for a re-review.

**Review by tevans78 — APPROVED** [review]

**Pull request merged**
